docs/attack-techniques/azure/azure.execution.vm-run-command.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Execute Commands on Virtual Machine using Run Command
+---
+
+# Execute Commands on Virtual Machine using Run Command
+
+ <span class="smallcaps w3-badge w3-orange w3-round w3-text-sand" title="This attack technique might be slow to warm up or detonate">slow</span> 
+ <span class="smallcaps w3-badge w3-blue w3-round w3-text-white" title="This attack technique can be detonated multiple times">idempotent</span> 
+
+Platform: Azure
+
+## MITRE ATT&CK Tactics
+
+- Execution
+
+## Description
+
+By utilizing the 'RunCommand' feature on a Virtual Machine, an attacker can pass:
+
+- Windows: PowerShell commands to the VM as SYSTEM.
+- Linux: Shell commands to the VM as root.
+
+References:
+
+- https://docs.microsoft.com/en-us/azure/virtual-machines/windows/run-command
+- https://docs.microsoft.com/en-us/azure/virtual-machines/linux/run-command
+- https://github.com/hausec/Azure-Attack-Matrix/blob/main/Execution/AZT201/AZT201-1.md
+
+<span style="font-variant: small-caps;">Warm-up</span>:
+
+- Create a virtual machine
+
+<span style="font-variant: small-caps;">Detonation</span>:
+
+- Invoke a RunCommand on the target virtual machine
+
+
+## Instructions
+
+```bash title="Detonate with Stratus Red Team"
+stratus detonate azure.execution.vm-run-command
+```
+
+## Detection
+
+1. Look for <code>Microsoft.Compute/virtualMachines/runCommand/action</code> events in Azure Activity logs
```

Comment:
The event I get in Azure logs isn't this one, it's `Microsoft.Compute/virtualMachines/runCommands/write`

Reply:
Also if I do it directly from the Azure portal UI, I get the event you said. 

- [ ] Make sure the audit log event generated matches the one when using the Azure console

Reply:
See [3a110ac](https://github.com/DataDog/stratus-red-team/pull/124/commits/3a110ac67634ec628a61d81eca651329687dd75b): I used a different API to run a command on the VM and it does trigger the same event (I had to reverse-engineer it from the API calls the Azure console makes...)

I think it makes the code slightly simpler also, what do you think?